dbdel.1.2

```diff
@@ -41,12 +42,7 @@ if [ -z "$1" ] || [ -z "$2" ]
 	echo "Error: can not read or write database file";
 	exit 1;
  else
-	NEWFILE=$(grep -v "^$1	" "$2");
-	if [ ! -z "$NEWFILE" ]
-	 then
-		echo "$NEWFILE" | cat > "$2" -;
-	 else
-		echo -n "" | cat > "$2" -;
-	fi;
+	grep -v "^$1	" "$2" >> "$2.temp";
```

Comment:
those five lines you saved needs to be used to detect file io collision. 'does db_file.temp exists'

Reply:
done and tested on debian